etna/transforms/math/differencing.py

```diff
@@ -207,15 +224,25 @@ def inverse_transform(self, df: pd.DataFrame) -> pd.DataFrame:
 
         Returns
         -------
-        result: pd.DataFrame
+        result:
             transformed DataFrame.
+
+        Raises
+        ------
+        ValueError:
+            if transform isn't fitted
+        NotImplementedError:
+            if there are segments that weren't present during training
         """
         if self._train_init_dict is None or self._test_init_df is None or self._train_timestamp is None:
-            raise AttributeError("Transform is not fitted")
+            raise ValueError("Transform is not fitted")
 
         if not self.inplace:
             return df
 
+        segments = df.columns.get_level_values("segment").unique().tolist()
```

Comment:
Why can we apply transform, but can't apply inverse_transform? May be we also need to diable transform for them?